### DISCUSSION TITLE
Support for shared feeds

### DISCUSSION THREAD

**Discussion by craigsteven89** [Athou/commafeed#1537] [Q&A] [answered]
Is there currently any support to allow shared feeds. For example being able to have the same feeds accessible by multiple users without having to add the feeds each time?

**Comment by Athou** [comment]
Nope that's not supported, but the feed will be fetched and stored only once for all users that are subscribed to it.

Users usually have different interests. What would be the use case of automatically subscribing multiple users to the same feed?

**Reply by craigsteven89** [reply]
Thanks for the response! I suspect my use case may be slightly niche, but I am a cyber security manager and I started using commafeed to keep updated on cyber news sources, threat intel articles etc. It would be useful to be able to share those sources with my team.

**Reply by Athou** [reply, marked as answer]
That would require a lot of work I think:
- introduce the concept of teams 
- introduce the concept of team leaders that can manage the feeds of their team members
- introduce feeds that cannot be unsubscribed from because they are managed by their team leader

All in all I think it adds a lot of complexity for a pretty niche use case like you said, especially for for a one-man project like CommaFeed. 

With the existing features of CommaFeed, I guess you could create an OPML file with the feeds you're following. You could then share that OPML file with your team and they would import it in their account.
